apps/src/lib/kits/maker/boards/microBit/MBFirmataWrapper.js

```diff
@@ -93,4 +93,10 @@ export default class MicrobitFirmataWrapper extends MBFirmataClient {
       callback(pin, this.digitalInput[pin] + 1);
     }
   }
+
+  compassCalibration() {
+    // Request that the micro:bit perform a compass calibration cycle
+    super.compassCalibration();
+    this.myPort.close();
```

Comment:
Do we need to wait for the compass calibration to complete before closing the close?